### PR TITLE
Implement EditWindow

### LINKED ISSUE #75: Implement EditWindow
Similar to `AddWindow`, have a new GUI window for editing users.

### PR DESCRIPTION
**Pull request by bakano98**
Implement `EditWindow`. This is a new screen that allows users to edit existing contacts in ModuleMate Finder. The UI is similar to that of `AddWindow`, with an additional `INDEX` field.
image

This is a basic implementation of `EditWindow`. It can be extended to also edit the `modules`s that is being taken by the user. Furthermore, `status` should also be able to be edited. This can be looked into another time.

**IMPORTANT**
Removed `assertParseFailure()` in `EditCommandParserTest` as having no index and no field being specified will no longer result in a `ParseFailure`, but will open up `EditWindow` instead.

Closes #75

### CODE REVIEW

File: src/main/resources/view/EditWindow.css [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+    -fx-background-color: #6F686D;
+}
+
+#emailLabel {
+    -fx-background-color: #6F686D;
+}
+
+#indexLabel {
+    -fx-background-color: #6F686D;
+}
```
Comment (lawwm) [inline src/main/resources/view/EditWindow.css:23]: Other than index, identical to AddWindow.css. Can merge into window.css?

**Review by lawwm — COMMENTED** [review]
LGTM

File: src/main/resources/view/EditWindow.css (hunk 1 above)
Comment (bakano98) [reply src/main/resources/view/EditWindow.css:23]: Good point. Will make the necessary changes sometime soon!

File: src/main/resources/view/EditWindow.fxml [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+<?import javafx.scene.control.Label?>
+<?import javafx.scene.control.TextField?>
+<?import javafx.scene.image.Image?>
+<?import javafx.scene.layout.HBox?>
+<?import javafx.scene.layout.Region?>
+<?import javafx.scene.layout.VBox?>
+<?import javafx.scene.text.Font?>
+<?import javafx.stage.Stage?>
+
+<fx:root resizable="false" title="Add Person" type="javafx.stage.Stage" xmlns="http://javafx.com/javafx/11" xmlns:fx="http://javafx.com/fxml/1">
```
Comment (Btaykb) [inline src/main/resources/view/EditWindow.fxml:16]: Should the title be `Edit Person` instead ?

**Review by Btaykb — COMMENTED** [review]
Overall LGTM, good work on extending the functionality of `edit`.

File: src/main/resources/view/EditWindow.fxml (hunk 2 above)
Comment (bakano98) [reply src/main/resources/view/EditWindow.fxml:16]: Yep, you are right. Will change that!

**Pull request merged**